### PR TITLE
Simplify hello and hello-dongle apps

### PR DESCRIPTION
**Pull request by jcorde**
Simplify sample apps and make their names more clear.

Depends on https://github.com/CESARBR/knot-documentation/pull/40

### CODE REVIEW

File: apps/blink/setup.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+CONFIG_BT_DEVICE_NAME="KNoT Blink"
```
Comment (ramonhpr) [inline apps/blink/setup.conf:1]: Use `git mv` to rename the file. This commit shows the previous file deleted and the new one added.
This way we keep the history of the file.

Comment (ramonhpr) [reply apps/blink/setup.conf:1]: Do the same in button sample

**Review by ramonhpr — CHANGES_REQUESTED** [review]
The patch is good. Just a change in the commit

File: apps/blink/setup.conf (hunk 1 above)
Comment (jcorde) [reply apps/blink/setup.conf:1]: It didn't work

**Pull request merged**